sdk/python/kfp/v2/dsl/component.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2021 Google LLC
```

Comment:
Do we want to leave this whole module file public?
This results in the following imports being available:
* `kfp.v2.dsl.component`
* `kfp.v2.dsl.component.component`
* `kfp.v2.dsl.component.components`

Reply:
No, the last two imports are not available.